src/core-plugins/CoreCommands.ts

```diff
@@ -83,7 +86,8 @@ export default class CoreCommands extends Plugin {
                 MessageActions.sendMessage(ctx.channel.id, {
                     content: `**Debug Info:**
                         > Discord: ${DebugInfo.discordVersion}
-                        > Aliucord: ${sha} (${Object.keys(plugins).length} plugins)
+                        > Aliucord: ${sha} (${Object.keys(plugins).length} plugins 
```

Comment:
```suggestion
                        > Aliucord: ${sha} (${Object.keys(plugins).length} plugins)
```

Reply:
LMAO I HATE GITHUB.DEV how did i remove that